packages/react-dom/src/events/__tests__/DOMPluginEventSystem-test.internal.js

```diff
@@ -1669,16 +1669,28 @@ describe('DOMPluginEventSystem', () => {
 
             function Test() {
               React.useEffect(() => {
-                setClick1(buttonRef.current, targetListener1);
-                setClick2(buttonRef.current, targetListener2);
-                setClick3(buttonRef.current, targetListener3);
-                setClick4(buttonRef.current, targetListener4);
+                const clearClick1 = setClick1(
```

Comment:
This test already had an error, but it was being swallowed.

@bvaughn made this same change in #19542, before it got reverted.